_posts/2025-06-18-hibernate7-on-quarkus.adoc

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: 'Hibernate 7 on Quarkus: every new version is a better database experience for the users'
+date: 2025-06-18
+tags: hibernate, database, jpa
+synopsis: a roundup on the new upgrades of Hibernate on Quarkus
+author: lmolteni
+---
+:imagesdir: /assets/images/posts/hibernate7
+ifdef::env-github,env-browser,env-vscode[:imagesdir: ../assets/images/posts/hibernate7]
+
+=== Introduction
+Hibernate is improving at a very fast speed, and so is its Quarkus support, as great database access is a key part of the Quarkus experience.
+The new Quarkus 3.24 release upgrades Hibernate to version 7, a major upgrade that implies some breaking changes, and thus will require some investigation with the https://docs.jboss.org/hibernate/orm/7.0/migration-guide/migration-guide.html[migration guide] when upgrading.
+Developers working on Hibernate and Quarkus are constantly collaborating, so here’s a quick peek at what has happened in the last months and what Quarkus users might expect in the future.
+
+=== License and Governance Updates 
+Both Quarkus and Hibernate are now projects of the https://www.commonhaus.org[Commonhaus Foundation], a non-profit organization dedicated to the sustainability of open-source libraries.
```

Comment:
Re-reading this, this kind of implies that our projects are not sustainable... which is obviously wrong. 

Maybe, still taking from their frontpage:

```suggestion
Both Quarkus and Hibernate are now projects of the https://www.commonhaus.org[Commonhaus Foundation], a non-profit organization dedicated to creating a collaborative environment for open-source libraries.
```